scipy/signal/spectral.py

```diff
@@ -402,7 +403,7 @@ def csd(x, y, fs=1.0, window='hann', nperseg=256, noverlap=None, nfft=None,
 
 def spectrogram(x, fs=1.0, window=('tukey',.25), nperseg=256, noverlap=None,
                 nfft=None, detrend='constant', return_onesided=True,
-                scaling='density', axis=-1, mode='psd'):
+                scaling='density', axis=-1):
```

Comment:
Sorry if I missed a discussion on this (?) -- this is backward incompatible, as it [exists in 0.18] (http://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.spectrogram.html).